.github/workflows/release-nym-vpn-x.yml

```diff
@@ -0,0 +1,452 @@
+name: release-nym-vpn-x
+on:
+  schedule:
+    - cron: "4 4 * * *"
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Tag name for release"
+        required: false
+        default: nightly-vpn-x
+      publish:
+        type: boolean
+        description: Publish Release
+        required: true
+        default: false
+  push:
+    tags:
+      - nym-vpn-x-v[0-9]+.[0-9]+.[0-9]+*
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04-16-core, macos-14]
+        target: [native]
+    runs-on: ${{ matrix.os }}
+    env:
+      UPDATER_BUNDLE_DIR: updater_bundle
+      APPIMAGE_DIR: appimage
+      WINDOWS_DIR: windows
+      DEB_DIR: deb
+    outputs:
+      pkg_version: ${{ steps.set_env.outputs.pkg_version }}
+      appimage_dir: ${{ env.APPIMAGE_DIR }}
+      windows_dir: ${{ env.WINDOWS_DIR }}
+      deb_dir: ${{ env.DEB_DIR }}
+
+    steps:
+      - name: "Cleanup working directory"
+        if: contains(matrix.os, 'custom')
+        shell: bash
+        run: |
+          ls -la ./
+          rm -rf ./* || true
+          rm -rf ./.??* || true
+          ls -la ./
+
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
```

Comment:
In the past this has bitten us were jobs reused old files, so imho let's skip sparse checkout

Reply:
these are the only nym deps
![image](https://github.com/nymtech/nym-vpn-client/assets/6359431/501c6b53-5a79-4fcf-92b9-171043156e5e)

so it's fine (should be)

Reply:
Well, should be, but there is also no performance advantage to doing a sparse checkout at least for our repo. So imho we should take the safe(r) route here

Reply:
ok forgot "should be", it's just fine

Reply:
imho let's remove the sparse please